Indicators/Stochastics.cs

```diff
@@ -56,27 +57,30 @@ public Stochastic(string name, int period, int kPeriod, int dPeriod)
             : base(name)
         {
             _maximum = new Maximum(name + "_Max", period);
-            _mininum = new Minimum(name + "_Min", period);
+            _minimum = new Minimum(name + "_Min", period);
             _sumFastK = new Sum(name + "_SumFastK", kPeriod);
             _sumSlowK = new Sum(name + "_SumD", dPeriod);
 
             FastStoch = new FunctionalIndicator<IBaseDataBar>(name + "_FastStoch",
                 input => ComputeFastStoch(period, input),
                 fastStoch => _maximum.IsReady,
-                () => _maximum.Reset()
+                ResetPrivateIndicators
```

Comment:
When the indicator's `Reset` method is called, the private indicators will be reset multiple times, 
I think an empty `Action` here should be fine: `() => { }` instead of `ResetPrivateIndicators`.

Same for `StochK` and `StochD` below.